README.md

```diff
@@ -1,2 +1,3 @@
 # test-v1
-This is a practice repo for understanding required git commnds.
+This is a practice repo for understanding required git commands.
+Making initial commit from local.
```

Comment:
Never mention anything about local.
Mention it as -> "Making initial commit from system"

Reply:
Addressed the comment